invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/list.overrides

```diff
@@ -54,6 +54,18 @@
       word-wrap: break-word;
       font-size: 80%;
     }
+
+    &.parent-doi:last-child {
+      padding-right: 0;
+      border-top: none;
+    }
+    &.parent-doi {
+      margin-top: .5rem;
+      word-wrap: break-word;
```

Comment:
these already exist for class .doi, see list.overrides line 50, could be reused?

Reply:
I tried it but display: block and font-size is not what we want for the parent doi....